addons/website/static/src/components/media_dialog/file_documents_selector.js

```diff
@@ -1,11 +0,0 @@
-import { patch } from "@web/core/utils/patch";
```

Comment:
Stable retrocompatibility: put back the old file with a comment explaining it's not used anymore but is kept due to the stable policy.